plugins/outputs/azure_data_explorer/sample.conf

```
@@ -23,3 +23,6 @@
   ## Creates tables and relevant mapping if set to true(default).
   ## Skips table and mapping creation if set to false, this is useful for running Telegraf with the lowest possible permissions i.e. table ingestor role.
   # create_tables = true
+
+  ##  Ingestion is supported through managed(streaming with fallback to batched ingestion or queued ) or queued, allowed values can be "managed" or "queued", with default to "queued"
+  # ingestion_type : "" 
```

Please follow the style found above!
```suggestion
  ##  Ingestion method to use.
  ##  Available options are
  ##    - managed  --  streaming with fallback to batched ingestion or the "queued" method below
  ##    - queued   --  queue up metrics and process sequentially
  # ingestion_type = "queued" 
```
Please check the documentation of the `queued` option (just a guess) and the default value!